diff --git a/node_modules/@oguzhnatly/react-native-image-manipulator/android/build.gradle b/node_modules/@oguzhnatly/react-native-image-manipulator/android/build.gradle
index 4a4983c..387fe52 100644
--- a/node_modules/@oguzhnatly/react-native-image-manipulator/android/build.gradle
+++ b/node_modules/@oguzhnatly/react-native-image-manipulator/android/build.gradle
@@ -2,7 +2,7 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
     }
 
     dependencies {
